src/vcpkg/binarycaching.cpp

```diff
@@ -92,7 +90,7 @@ namespace
             std::string segment;
             for (;;)
             {
-                auto n = match_until([](char32_t ch) { return ch == ',' || ch == '`' || ch == ';'; });
+                auto n = match_while([](char32_t ch) { return ch != ',' && ch != '`' && ch != ';'; });
```

Comment:
why this change?

Reply:
To merge match_until and match_zero_or_more into one function.

Reply:
Reverted.